Libraries/Animated/nodes/AnimatedProps.js

```diff
@@ -10,16 +10,18 @@
 
 'use strict';
 
+import typeof AnimatedNodeType from './AnimatedNode';
 import type {PlatformConfig} from '../AnimatedPlatformConfig';
 
-const ReactNative = require('../../Renderer/shims/ReactNative');
-const {AnimatedEvent} = require('../AnimatedEvent');
-const NativeAnimatedHelper = require('../NativeAnimatedHelper');
-const AnimatedNode = require('./AnimatedNode');
-const AnimatedStyle = require('./AnimatedStyle');
-const invariant = require('invariant');
+import ReactNative from '../../Renderer/shims/ReactNative';
+import {AnimatedEvent} from '../AnimatedEvent';
+import * as NativeAnimatedHelper from '../NativeAnimatedHelper';
+import invariant from 'invariant';
 
-class AnimatedProps extends AnimatedNode {
+const AnimatedNode: AnimatedNodeType = require('./AnimatedNode').default;
+const AnimatedStyle = require('./AnimatedStyle').default;
```

Comment:
Couldn't get jest tests to pass using import on these components

Reply:
Managed to replace these with imports by updating `Animated-test` to require `AnimatedProps` instead of importing